_docs/concepts/traffic-management/overview.md

```diff
@@ -1,44 +1,36 @@
 ---
 title: Overview
-overview: Provides a conceptual overview of traffic management principles in Istio and the kind of features enabled by these principles.
+overview: Provides a conceptual overview of traffic management in Istio and the features enabled by its traffic management principles.
                 
 order: 0
 
 layout: docs
 type: markdown
 ---
 
-Existing container orchestration platforms like Kubernetes, and Mesos, and
-other microservice frameworks allow operators to control when a particular
-set of pods/VMs should receive traffic (e.g., by adding/removing specific
-labels). Unlike existing techniques, Istio decouples traffic flow and infrastructure
-scaling. 
+This page provides an overview of how traffic management works in Istio, including the benefits of its traffic management principles. It assumes that you've already read [What Is Istio?]({{home}}/docs/concepts/what-is-istio/overview.html) and are familiar with Istio's high-level architecture. You can find out more about individual traffic management services and features in the other guides in this section.
+
+## Istio-Manager and Envoy
+
+The core component used for traffic management in Istio is the [Istio-Manager]({{home}}/docs/concepts/traffic-management/manager.html) service, which manages and configures all the Envoy proxy instances deployed in a particular Istio service mesh. It lets you specify what rules you want to use to route traffic between Envoy proxies and configure failure recovery features such as timeouts. It also maintains a canonical model of all the services in the mesh and uses this to let Envoys know about the other instances in the mesh via its discovery service.
+
+Each Envoy instance maintains [load balancing information]({{home}}/docs/concepts/traffic-management/load-balancing.html) based on the information it gets from Istio-Manager and periodic health-checks of other instances in its load-balancing pool, allowing it to intelligently distribute traffic between destination instances while following its specified routing rules.
+
+## Traffic management benefits
```

Comment:
How about changing it to "Traffic management principles"?

(Also obviously it is our philosophy, but it *provides* benefits, and I think that's what users are going to care about. IME, users are generally not particularly interested, even in a conceptual overview, about why development teams made particular design decisions or their abstract design goals, they just want to know how the thing works and why they should use it - it's what distinguishes a user-facing conceptual doc from a design doc or RFC. I know we're also trying to recruit contributors but even still...)